CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+## 2.5.0-rc.0 / 2018-10-23
+
+Starting with the s390x platform for Linux is officially supported.
+
+* [CHANGE] Remote write default for a shards buffer capacity was changed from `100000` to `10000`.
+* [CHANGE] Remote write default for a retries was changed from `10` to `3`.
+* [FEATURE] Add OpenMetrics support (EXPERIMENTAL).
+* [FEATURE] Add unit testing for rules.
+* [ENHANCEMENT] Make maximum number of samples per query configurable via `--query.max-samples` flag.
+* [ENHANCEMENT] Make maximum number of concurrent remote reads configurable via `--storage.remote.read-concurrent-limit` flag.
+* [ENHANCEMENT] Remote Read: Add `prometheus_api_remote_read_queries` metric tracking currently executed or waiting remote read queries.
+* [ENHANCEMENT] Remote Read: Reduced memory usage.
+* [ENHANCEMENT] Discovery: Split `prometheus_sd_discovered_targets` metric by scrape and notify as well as by section in the respective configuration.
+* [ENHANCEMENT] Discovery: Add `prometheus_sd_discovered_targets`, `prometheus_sd_received_updates_total`, `prometheus_sd_updates_delayed_total`, and `prometheus_sd_updates_total` metrics for discovery subsystem.
+* [ENHANCEMENT] Discovery: Improve performance of previously slow updates of changes of targets.
+* [ENHANCEMENT] Kubernetes SD: Add extended metrics.
+* [ENHANCEMENT] OpenStack SD: Support discovering instances from all projects.
+* [ENHANCEMENT] OpenStack SD: Discover all interfaces.
+* [ENHANCEMENT] OpenStack SD: Support `tls_config` for the used HTTP client.
+* [ENHANCEMENT] EC2 SD: Expose owner of an instance as meta label.
+* [ENHANCEMENT] Triton SD: Add ability to filter triton_sd targets by pre-defined groups.
+* [ENHANCEMENT] Web UI: Avoid browser spell-checking in expression field.
+* [ENHANCEMENT] Web UI: Add scrape duration and last evaluation time in targets and rules pages.
+* [ENHANCEMENT] Web UI: Improve rule view by wrapping lines.
+* [BUGFIX] Fix histograms on 32 bit platforms.
+* [BUGFIX] Change max/min over_time to handle NaNs properly.
+* [BUGFIX] Discovery: Properly unregister source when targets are empty.
```

Comment:
This bug fix was in the file SD adapter code so it probably doesn't warrant an entry.